CHANGELOG.md

```diff
@@ -60,6 +60,8 @@ List of incompatible ABI changes in this release:
 * avifEncoderSetCodecSpecificOption() now returns avifResult instead of void to
   report memory allocation failures.
 * At decoding, avifIOStats now returns the same values as at encoding.
+* avifGetPixelFormatInfo() now returns 0 for chromaShiftX and chromaShiftY when
+  the format is AVIF_PIXEL_FORMAT_YUV400. Previously it returned 1.
```

Comment:
This is how they're coded in the AV1 bitstream. There is one note about MC_IDENTITY and subsampling=0:

https://aomediacodec.github.io/av1-spec/#color-config-syntax
https://aomediacodec.github.io/av1-spec/#color-config-semantics

Reply:
> There is one note about MC_IDENTITY

Indeed:

> If matrix_coefficients is equal to MC_IDENTITY, it is a requirement of bitstream conformance that subsampling_x is equal to 0 and subsampling_y is equal to 0.

This contradicts the table:

> subsampling_x	| subsampling_y |	mono_chrome |	Description
>-|-|-|-
> 1	|1|	1|	Monochrome 4:0:0

Unless MC_IDENTITY is forbidden for 4:0:0?

Reply:
> Unless MC_IDENTITY is forbidden for 4:0:0?

Yes it is. You can see that in the color config syntax where monochrome is treated separately and in the case of MC_IDENTITY subsampling is forced to 0.

Reply:
James: The part of the color config syntax that forces subsampling to 0 in the case of MC_IDENTITY is the following:

```
    } else if ( color_primaries == CP_BT_709 &&
                transfer_characteristics == TC_SRGB &&
                matrix_coefficients == MC_IDENTITY ) {
        color_range = 1
        subsampling_x = 0
        subsampling_y = 0
    } else {
```

The question is: what happens if color_primaries == CP_BT_709, transfer_characteristics == TC_SRGB, matrix_coefficients == MC_IDENTITY, and mono_chrome == 1. In this case, subsampling is forced to 1 because mono_chrome is 1, and I think we will go by the following and declare the bitstream as not conforming to the spec:

> If matrix_coefficients is equal to MC_IDENTITY, it is a requirement of bitstream conformance that subsampling_x is equal to 0 and subsampling_y is equal to 0.